src/Articles.jsx

```diff
@@ -0,0 +1,41 @@
+import {useEffect,useState} from 'react'
+import { getArticles } from './api';
+import {Link} from 'react-router-dom';
+
+const Articles = () => {
+    const [articleList, setArticleList] = useState([])
+    const [isLoading, setIsLoading] = useState(true)
+
+    useEffect(() => {
+        setIsLoading(true)
+        getArticles()
+        .then((data) => {
+            setArticleList(data)
+            setIsLoading(false)
+        })
+    },[])
+
+   if(isLoading) {
+    return <p className='Loading'>Loading...</p>;
+   }
+
+
+    return (
+        <div className="article-list">
```

Comment:
Is there a more appropriate semantic HTML tag which would work here? Take a look at: [https://www.w3schools.com/html/html5_semantic_elements.asp](url)